Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -2,7 +2,7 @@ import UIKit
 
 /// Renders a receipt in an AirPrint enabled printer.
 /// To be properly implemented in https://github.com/woocommerce/woocommerce-ios/issues/3978
-final class ReceiptRenderer: UIPrintPageRenderer {
+public final class ReceiptRenderer: UIPrintPageRenderer {
```

Comment:
As I mentioned in the PR description, we could probably extract what htmlContent() does to a separate class, and make that new class public instead of this one